heatmap.R

```diff
@@ -50,3 +50,5 @@ legend("topright",legend=c("Ornamental","Perennial","Annual","Shrub","Tree","Gra
        fill=c("red","blue","cyan","pink","yellow","green","white","white","grey","black","white","darkorchid","darkred","white","green","darkgreen"), border=FALSE, bty="n", y.intersp = 0.7, cex=0.7)
 dev.off()
 
+# note
```

Comment:
Thanks for tying this out. Here is where I can make comments about your changes and we can have a dialogue about what was done. 
Obviously we do not need to incorporate the 'note' right now. The code seems to be working fine without it. :)